lib/tax_cloud/address.rb

```diff
@@ -15,6 +15,8 @@ class Address < Record
     attr_accessor :zip5
     # 4-digit Zip Code.
     attr_accessor :zip4
+    # RDI
```

Comment:
Nit: let's expand what RDI means here?

Reply:
I suppose it's `Residential Delivery Indicator`